sympy/matrices/dense.py

```diff
@@ -1281,16 +1281,16 @@ def hessian(f, varlist, constraints=[]):
     return out
 
 
-def jordan_cell(eigenval, n):
+def jordan_block(eigenval, n):
```

Comment:
We need to include the deprecated version of this function as well and wrap it with the deprecation decorator.